src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/probation/DeliusApiClient.kt

```diff
@@ -33,10 +33,9 @@ class DeliusApiClient(@Qualifier("oauthDeliusApiClient") val communityApiClient:
 
   fun getStaffDetailsByUsername(username: String): User? {
     val communityApiResponse = communityApiClient
```

Comment:
updated to get method as we need to fetch only one user(staff details) now.
Refer [swagger](https://create-and-vary-a-licence-and-delius-dev.hmpps.service.justice.gov.uk/swagger-ui/index.html#/staff-resource/findStaff).